package.json

```diff
@@ -480,23 +480,23 @@
     "@playwright/test": "^1.39.0",
     "@pmmmwh/react-refresh-webpack-plugin": "^0.5.11",
     "@sentry/cli": "^2.19.4",
-    "@storybook/addon-a11y": "^7.6.19",
-    "@storybook/addon-actions": "^7.6.19",
+    "@storybook/addon-a11y": "^7.6.20",
+    "@storybook/addon-actions": "^7.6.20",
     "@storybook/addon-designs": "^7.0.9",
-    "@storybook/addon-docs": "^7.6.19",
-    "@storybook/addon-essentials": "^7.6.19",
+    "@storybook/addon-docs": "^7.6.20",
+    "@storybook/addon-essentials": "^7.6.20",
     "@storybook/addon-knobs": "^7.0.2",
-    "@storybook/addon-mdx-gfm": "^7.6.19",
-    "@storybook/addons": "^7.6.19",
-    "@storybook/api": "^7.6.19",
-    "@storybook/client-api": "^7.6.19",
-    "@storybook/components": "^7.6.19",
-    "@storybook/core-events": "^7.6.19",
-    "@storybook/react": "^7.6.19",
-    "@storybook/react-webpack5": "^7.6.19",
+    "@storybook/addon-mdx-gfm": "^7.6.20",
+    "@storybook/addons": "^7.6.20",
+    "@storybook/api": "^7.6.20",
+    "@storybook/client-api": "^7.6.20",
+    "@storybook/components": "^7.6.20",
+    "@storybook/core-events": "^7.6.20",
+    "@storybook/react": "^7.6.20",
+    "@storybook/react-webpack5": "^7.6.20",
     "@storybook/storybook-deployer": "^2.8.16",
     "@storybook/test-runner": "^0.14.1",
-    "@storybook/theming": "^7.6.19",
+    "@storybook/theming": "^7.6.20",
```

Comment:
`@storybook/*` version bumps cherry-picked from https://github.com/MetaMask/metamask-extension/pull/26626 to avoid mixed use of `7.6.19` and `7.6.20`.

Reply:
@legobeat I cherry-picked this over to here since it looks like this'll be merged first.

Reply:
Makes sense!